Lib/AuthContainer.php

```diff
@@ -1,8 +0,0 @@
-<?php
```

Comment:
Still not sure whether this is usefull or not. Could be used for typehinting in dependent applications.